pipelines/skylab/optimus/Optimus.changelog.md

```diff
@@ -1,3 +1,10 @@
+# 4.0.1
+
+2020-09-14 (Date of Last Commit)
+
+* Replaced the tasks FastqToUBam, AttachBarcodes, SplitBamByCellBarcode, ScatterBam with a new task FastqProcessing
```

Comment:
```suggestion
* Refactored the pipeline to preprocess fastqs using the task `FastqProcessing`. Outputs are identical and the pipeline should be significantly faster
```

I'm not going to claim this is a better changelog entry than the original, I just wanted to clearly note no changes to outputs and the speed improvements @kishorikonwar @ekiernan thoughts?